generators/server/templates/_pom.xml

```diff
@@ -1287,7 +1287,7 @@
                         <artifactId>maven-war-plugin</artifactId>
                         <configuration>
                             <%_ if (!skipClient) { _%>
-                            <warSourceDirectory><%= CLIENT_MAIN_SRC_DIR %></warSourceDirectory>
+                            <warSourceDirectory><%= CLIENT_DIST_DIR %></warSourceDirectory>
```

Comment:
Oh yeah, I'm using Angular 2 which require transpiling. I will add a check for angular 2